targets/TARGET_Silicon_Labs/TARGET_EFM32/watchdog_api.c

```diff
@@ -93,6 +93,20 @@ watchdog_features_t hal_watchdog_get_platform_features(void)
         .max_timeout = 262145,
         .update_config = true,
         .disable_watchdog = true
+    #if defined(TARGET_EFM32GG) || defined(TARGET_EFM32HG) || defined(TARGET_EFM32LG) || defined(TARGET_EFM32WG) || defined(TARGET_EFM32ZG)
+        .clock_typical_frequency = 1000;
+        .clock_max_frequency = 1750;
+    #elif defined(TARGET_EFM32GG11)
+        .clock_typical_frequency = 1000;
+        .clock_max_frequency = 1120;
+    #elif defined(TARGET_EFM32PG) || defined(TARGET_EFM32PG12) || defined(TARGET_EFR32MG1) || defined(TARGET_EFR32MG12)
+        .clock_typical_frequency = 1000;
+        .clock_max_frequency = 1070;
+    #else
+        //unspecified target, assume it's perfect.
```

Comment:
Let's stay consistent with other targets.
```suggestion
#error "unsupported target" 
```